flexget/utils/cached_input.py

```diff
@@ -165,7 +165,8 @@ def wrapped_func(*args, **kwargs):
                     self.cache[cache_name] = copy.deepcopy(response)
                 except TypeError:
                     # might be caused because of backlog restoring some idiotic stuff, so not neccessarily a bug
-                    log.critical('Unable to save task content into cache, if problem persists longer than a day please report this as a bug')
+                    log.critical('Unable to save task content into cache, if problem persists longer than a day \
```

Comment:
This adds \ into might with a lot empty spaces afterwards.

function('fkgivjfbf'
                'gkvkfbg')

Is best way to do this